upgrade/patches/43_api_token.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE {{%api_token}} (
+  apt_id int unsigned NOT NULL auto_increment,
+  apt_usr_id int unsigned NOT NULL,
+  apt_created datetime NOT NULL,
+  apt_status varchar(10) NOT NULL default 'active',
+  apt_token varchar(32) NOT NULL,
+  PRIMARY KEY (apt_id),
+  KEY (apt_token)
```

Comment:
Ah, sorry I always think of KEY just as "primary key" and INDEX for other keys.  I'll add it.

Your comment about mysqladmin makes sense for people on shared hosts, etc and I'll add that.